harness/propertyHelper.js

```diff
@@ -268,3 +269,32 @@ function verifyNotConfigurable(obj, name) {
     throw new Test262Error("Expected obj[" + String(name) + "] NOT to be configurable, but was.");
   }
 }
+
+/**
+ * Use this function to verify the properties of a primordial object.
+ * verifyPrimordialProperty accounts for the expected differences in the writable and configurable attributes when running under Hardened JavaScript's lockdown.
```

Comment:
for example, there are no compliant differences allowed in these attributes. writability and configurability is strictly defined by the spec.

Reply:
@ljharb – Please read the full thread, specifically:

> Sure, until the relevant Hardened JavaScript proposals make their way to Stage 2.7, let's keep the Hardened JavaScript specific harness changes outside the test262 main branch.

Reply:
ah ok, well then that'll be great once implemented :-)